src/cr/cube/crunch_cube.py

```diff
@@ -1177,6 +1134,32 @@ def _margin_pruned_indices(margin, inserted_ind, axis):
 
         return pruned_ind
 
+    def _measure(self, weighted):
+        """_BaseMeasure subclass representing primary measure for this cube.
+
+        If the cube response includes a means measure, the return value is
+        means. Otherwise it is counts, with the choice between weighted or
+        unweighted determined by *weighted*.
+
+        Note that weighted counts are provided on an "as-available" basis.
+        When *weighted* is True and the cube response is not weighted,
+        unweighted counts are returned.
+        """
+        return (
+            self._measures.means if self._measures.means is not None else
```

Comment:
Maybe the explicit form would be more readable here:
```
measures = self._measures
if measures.means is not None:
    return measures.means
if weighted:
    return measures.weighted_counts
return measures.unweighted_counts
```
I believe it would also not hide potential misses by coverage (not sure about this though). I tried it locally, and it increases the unit test coverage from 65.94% to 66.10%